kura/org.eclipse.kura.linux.bluetooth/src/main/java/org/eclipse/kura/linux/bluetooth/le/BluetoothGattImpl.java

```diff
@@ -276,6 +278,51 @@ public void processInputStream(String string) {
     public void processErrorStream(String string) {
     }
 
+    @Override
+    public BluetoothGattSecurityLevel getSecurityLevel() throws KuraException {
+
+        BluetoothGattSecurityLevel level = BluetoothGattSecurityLevel.UNKNOWN;
+        if (this.m_connected && this.m_proc != null) {
+            this.m_securityLevel = "";
```

Comment:
If securityLevel is of type BluetoothGattSecurityLevel, I can't discriminate if the getSecurityLevel returns an error, timeout or a valid result. So, I left the code unchanged.